tests/unit/ml_handlers/data/house_sales.csv

```diff
@@ -1,348 +1,348 @@
-2016-06-30,324725,unit,1
-2016-09-30,325127,unit,1
-2016-12-31,325521,unit,1
-2017-03-31,327870,unit,1
-2017-06-30,330319,unit,1
-2017-09-30,332481,unit,1
-2017-12-31,334804,unit,1
-2018-03-31,336637,unit,1
-2018-06-30,338105,unit,1
-2018-09-30,339220,unit,1
-2018-12-31,339350,unit,1
-2019-03-31,337838,unit,1
-2019-06-30,336551,unit,1
-2019-09-30,335449,unit,1
-2007-06-30,368817,unit,2
-2007-09-30,368817,unit,2
-2007-12-31,368817,unit,2
-2008-03-31,368817,unit,2
-2008-06-30,373482,unit,2
-2008-09-30,377481,unit,2
-2008-12-31,382010,unit,2
-2009-03-31,380810,unit,2
-2009-06-30,385791,unit,2
-2009-09-30,391161,unit,2
-2009-12-31,396448,unit,2
-2010-03-31,402898,unit,2
-2010-06-30,408608,unit,2
-2010-09-30,412509,unit,2
-2010-12-31,415991,unit,2
-2011-03-31,417970,unit,2
-2011-06-30,419777,unit,2
-2011-09-30,421158,unit,2
-2011-12-31,423144,unit,2
-2012-03-31,424673,unit,2
-2012-06-30,424249,unit,2
-2012-09-30,425453,unit,2
-2012-12-31,425922,unit,2
-2013-03-31,425751,unit,2
-2013-06-30,426621,unit,2
-2013-09-30,428398,unit,2
-2013-12-31,428365,unit,2
-2014-03-31,429283,unit,2
-2014-06-30,429361,unit,2
-2014-09-30,428911,unit,2
-2014-12-31,429832,unit,2
-2015-03-31,431567,unit,2
-2015-06-30,432730,unit,2
-2015-09-30,432791,unit,2
-2015-12-31,432801,unit,2
-2016-03-31,431418,unit,2
-2016-06-30,430880,unit,2
-2016-09-30,430654,unit,2
-2016-12-31,430308,unit,2
-2017-03-31,429897,unit,2
-2017-06-30,429059,unit,2
-2017-09-30,428878,unit,2
-2017-12-31,428532,unit,2
-2018-03-31,427856,unit,2
-2018-06-30,427623,unit,2
-2018-09-30,426970,unit,2
-2018-12-31,426936,unit,2
-2019-03-31,426669,unit,2
-2019-06-30,425659,unit,2
-2019-09-30,424412,unit,2
-2007-09-30,518911,unit,3
-2007-12-31,518911,unit,3
-2008-03-31,518911,unit,3
-2008-06-30,518911,unit,3
-2008-09-30,518911,unit,3
-2008-12-31,518911,unit,3
-2009-03-31,518911,unit,3
-2009-06-30,518911,unit,3
-2009-09-30,523285,unit,3
-2009-12-31,522862,unit,3
-2010-03-31,524008,unit,3
-2010-06-30,535063,unit,3
-2010-09-30,538694,unit,3
-2010-12-31,555117,unit,3
-2011-03-31,550851,unit,3
-2011-06-30,547981,unit,3
-2011-09-30,539828,unit,3
-2011-12-31,530987,unit,3
-2012-03-31,540344,unit,3
-2012-06-30,537592,unit,3
-2012-09-30,548326,unit,3
-2012-12-31,555644,unit,3
-2013-03-31,566706,unit,3
-2013-06-30,580696,unit,3
-2013-09-30,581428,unit,3
-2013-12-31,586470,unit,3
-2014-03-31,583883,unit,3
-2014-06-30,583370,unit,3
-2014-09-30,598512,unit,3
-2014-12-31,598812,unit,3
-2015-03-31,599507,unit,3
-2015-06-30,602877,unit,3
-2015-09-30,603343,unit,3
-2015-12-31,612295,unit,3
-2016-03-31,617363,unit,3
-2016-06-30,622045,unit,3
-2016-09-30,616198,unit,3
-2016-12-31,610618,unit,3
-2017-03-31,606935,unit,3
-2017-06-30,605273,unit,3
-2017-09-30,606850,unit,3
-2017-12-31,604413,unit,3
-2018-03-31,604293,unit,3
-2018-06-30,603434,unit,3
-2018-09-30,603281,unit,3
-2018-12-31,601167,unit,3
-2019-03-31,605637,unit,3
-2019-06-30,599339,unit,3
-2019-09-30,597884,unit,3
\ No newline at end of file
+saledate,ma,type,bedrooms
```

Comment:
Hmm I wonder what is going on with the diff on this file, can you please revert?

Reply:
looks like it some how went thru linting which added a line add end of the csv.. (all resolved now)